tests/new_test.go

```diff
@@ -26,95 +30,6 @@ func deleteIfNotFailed(e *ptesting.Environment) {
 }
 
 func TestPulumiNew(t *testing.T) {
-	t.Run("SanityTest", func(t *testing.T) {
```

Comment:
I haven't had a chance to fix-up these existing tests to play well in the new git-based model. When I get a chance, I should be able to add back most of these cases.